pom.xml

```diff
@@ -4,7 +4,7 @@
   <parent>
     <groupId>org.jenkins-ci.plugins</groupId>
     <artifactId>plugin</artifactId>
-    <version>1.554.2</version>
+    <version>1.565</version>
```

Comment:
Yes, we need the doReload()-Method in PromotionContextExtensionPoint to to update the Item.